### PR TITLE
Adding a servlet to fetch accessories that the player has equipped

### PR DESCRIPTION
**Pull request by lzoebaker**
The servlet will respond with a JSON object formatted as follows:
"equippedHat": (hat accessory object), 
"equippedGlasses": (glasses accessory object),
"equippedCompanion": (companion accessory object)

If the user does not have an accessory equipped or for some edge case reason the accessory object is not found in the database and a EntityNotFoundException is thrown, the object will be replaced by the string "noneEquipped."

This JSON string will be parsed in the front end to create the accessories on the player.

### CODE REVIEW

File: src/test/java/com/google/sps/GetEquippedAccessoriesTest.java [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+  public void testThatServletRespondsWithEquippedAccessories() throws IOException {
+    StringWriter stringWriter = new StringWriter();
+    PrintWriter printWriter = new PrintWriter(stringWriter);
+    when(this.response.getWriter()).thenReturn(printWriter);
+
+    User user = this.localUserService.getCurrentUser();
+    List<String> accessories = ALL_ACCESSORIES;
+    String equippedHatID = HAT_ID;
+    String equippedGlassesID = GLASSES_ID;
+    String equippedCompanionID = COMPANION_ID;
```
Comment (menynexi) [inline src/test/java/com/google/sps/GetEquippedAccessoriesTest.java:126]: where are these variables being used ?

File: src/test/java/com/google/sps/GetEquippedAccessoriesTest.java [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+    createCurrentPlayerAndAddToDatabase(user, ALL_ACCESSORIES, HAT_ID, GLASSES_ID, COMPANION_ID);
```
Comment (menynexi) [inline src/test/java/com/google/sps/GetEquippedAccessoriesTest.java:127]: shouldn't they be used here

Comment (lzoebaker) [reply src/test/java/com/google/sps/GetEquippedAccessoriesTest.java:127]: good catch, thanks many!

File: src/test/java/com/google/sps/GetEquippedAccessoriesTest.java (hunk 1 above)
Comment (lzoebaker) [reply src/test/java/com/google/sps/GetEquippedAccessoriesTest.java]: changed :)

File: src/test/java/com/google/sps/GetEquippedAccessoriesTest.java [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+    this.getEquippedAccessories.init();
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  @Test
+  public void testThatServletRespondsWithEquippedAccessories() throws IOException {
```
Comment (WillHipschman) [inline src/test/java/com/google/sps/GetEquippedAccessoriesTest.java:117]: name tests "method_condition_result"

**Pull request merged**